src/python/pants/notes/2.10.x.md

```diff
@@ -1,5 +1,67 @@
 # 2.10.x Release Series
 
+## 2.10.0.dev2 (Jan 22, 2022)
+
+### New Features
+
+* JVM lockfile invalidation headers and verification ([#14185](https://github.com/pantsbuild/pants/pull/14185))
+
+* Add new `api-types` help goal ([#14081](https://github.com/pantsbuild/pants/pull/14081))
+
+* Add line numbers to `--unowned-dependency-behavior` errors ([#14169](https://github.com/pantsbuild/pants/pull/14169))
+
+* Add Flake8 first-party plugin support ([#14027](https://github.com/pantsbuild/pants/pull/14027))
+
+* add runtime dependencies field for Apache Thrift for Python ([#14158](https://github.com/pantsbuild/pants/pull/14158))
+
+* Add `#pants: ignore` pragma for ignoring imports (note: This was renamed to no-infer-dep) ([#14059](https://github.com/pantsbuild/pants/pull/14059))
+
+* Rename `# pants: ignore` to `# pants: no-infer-dep` ([#14225](https://github.com/pantsbuild/pants/pull/14225))
```

Comment:
If you take the above recommendation, you could probably remove this. Because it all happened inside a single dev release, there was no public API change etc.